README.md

```diff
@@ -96,6 +102,20 @@ if (closed_issues.length) {
 }
 ```
 
+### Direct usage (not as a plugin)
+
+```js
+const octokit = new Octokit({ auth: "secret123" });
+
+const { data, updated } = await composeUniqueIssue(octokit, {
```

Comment:
...`composeUniqueIssue` is fairly different from the `.createOrUpdateUniqueIssue` method name added by the plugin. which is better to align the naming with for this addition?